app/models/allocation.rb

```diff
@@ -6,5 +6,6 @@ class Allocation < ApplicationRecord
     :nomis_booking_id,
     :prison,
     :allocated_at_tier,
+    :responsibility,
```

Comment:
I think the problem with storing it in the allocation, is that it was the responsibility at the time of the allocation, so the 10.months problem won't be reflected automatically (because we're always recalculating based on the current date).  This might be okay, but it depends on whether the expectation is that the responsibility will automatically change on a specific day.

Maybe it's okay for now, but we should raise a ticket to clarify as part of the primary/secondary allocation work we will need to do.

Reply:
Yes good point - I guess it would be good to know whether a new allocation is required if an Offenders sentence hits the < 10 month mark?